tests/unit/Option/SomeTest.php

```diff
@@ -81,6 +82,19 @@ public function testContains(): void
         static::assertTrue($option->contains(2));
     }
 
+    public function testApply(): void
+    {
+        $spy = new Ref(1);
+
+        $option = Option\some(2);
+        $option->apply(static function (int $value) use ($spy) {
```

Comment:
can you add the actual check here as well?

Reply:
Not 100% sure if I understood what you mean, I've moved those `2` values to a variable called `$actual`

Reply:
No, I meant checking the return value of apply here as well. As how you did it in the None test.

Reply:
Sorry, gotcha! 👍🏻